src/export-plan/pdf-gen.ts

```diff
@@ -0,0 +1,328 @@
+import jsPDF from 'jspdf';
+import autoTable from 'jspdf-autotable';
+import {
+  getCollegeFullName,
+  getMajorFullName,
+  getMinorFullName,
+  sortedSemesters,
+  isPlaceholderCourse,
+  getGradFullName,
+} from '../utilities';
+import store from '../store';
+import { addFonts } from './add-fonts';
+import { getCollegeAbbr } from '@/requirements/data/index';
```

Comment:
`@` often cause a lot of issues with running typescript in the codebase for some reason
```suggestion
import { getCollegeAbbr } from '../requirements/data';
```